scripts/network/prepare-tap.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+# This script helps to prepare an environment for developing berserker network
+# workload. There are following preparation steps:
+#   * Create and start up a new tun device for berserker to use
+#   * Optionally prepare iptables for the device to be visible
+#
+# The last step is optional, because iptables configuration could be different
+# between development environments. Meaning it's not guaranteed this part of
+# the script is suitable for every case.
+
+stop() { echo "$*" 1>&2 ; exit 1; }
+
+which ip &>/dev/null || stop "Don't have the ip tool"
+which whoami &>/dev/null || stop "Don't have the whoami tool"
+which iptables &>/dev/null || stop "Don't have the iptables tool"
+which sysctl &>/dev/null || stop "Don't have the sysctl tool"
+
+ADDRESS="192.168.0.1/16"
+NAME="tun0"
+USER="`whoami`"
+CONFIGURE_IPTABLE="false"
+CONFIGURE_TUNTAP_IF_EXISTS="false"
+
+while getopts ":a:t:uio" opt; do
```

Comment:
```suggestion
while getopts ":a:t:u:io" opt; do
```
I think u requires an argument so there should be a ":" after it.